--- a/python/RestAuthCommon/handlers.py
+++ b/python/RestAuthCommon/handlers.py
@@ -341,7 +341,11 @@
         return self.library.dumps({'l': self.normalize_list(obj), })
 
     def marshal_str(self, obj):
-        return self.library.dumps({'s': self.normalize_str(obj), })
+        try:
+            return self.library.dumps({'s': self.normalize_str(obj), })
+        except AttributeError as e:
+            print(self.library, sorted(dir(self.library)))
+            raise e
 
     def unmarshal_dict(self, body):
         if isinstance(body, unicode):
